gtda/images/filtrations.py

```diff
@@ -618,3 +624,150 @@ def transform(self, X, y=None):
         Xt = np.concatenate(Xt)
 
         return Xt
+
+
+@adapt_fit_transform_docs
+class SignedDistanceFiltration(BaseEstimator, TransformerMixin):
+    """Filtrations of 2D/3D binary images based on the dilation and the erosion
+    of activated regions.
+
+    This filtration assigns to each pixel in an image a grayscale value
+    calculated as follows. For activated pixels, if the minimum Manhattan
```

Comment:
Are you sure that "For activated pixels" shouldn't be "For deactivated pixels", and vice-versa later?